tools/dependency/release_dates.py

```diff
@@ -38,32 +39,72 @@ def VerifyAndPrintLatestRelease(dep, repo, metadata_version, release_date):
   try:
     latest_release = repo.get_latest_release()
     if latest_release.created_at > release_date and latest_release.tag_name != metadata_version:
-      print(f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
-            f'{latest_release.tag_name}@<{latest_release.created_at}>')
-  except github.UnknownObjectException:
+      print(Fore.YELLOW +
+            f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}> : '
+            f'{latest_release.tag_name}@<{latest_release.created_at}>' + Style.RESET_ALL)
+  # except github.UnknownObjectException:
+  except github.GithubException as e:
+    # print(Fore.RED + 'GitHub API status get_latest_release(): {}'.format(str(e)) + Style.RESET_ALL)
```

Comment:
Debug remnant? The `except` catch is quite general, should we revert back to the more specific `UnknownObjectException`?

Reply:
Done